config/routes.rb

```diff
@@ -20,18 +20,25 @@
   get '/artists/:id' => 'artists#show', as: :artistid
   get '/artists/', to: 'artists#all_artists'
 
+  get '/transactions/' => 'transactions#show'
```

Comment:
Typically when you want to retrieve all entries in a db, it is an index method so we should rename this to transactions#index. show is normally used for a single entry

Also, can you add a link to the transactions within the header so that upon hovering over artist name you can click on transactions? You should probably be able to pattern-match based on what's already there but if you have q's you should ask me